flink-core/src/main/java/org/apache/flink/configuration/NettyShuffleEnvironmentOptions.java

```diff
@@ -234,6 +234,14 @@
 			.withDeprecatedKeys("taskmanager.net.client.connectTimeoutSec")
 			.withDescription("The Netty client connection timeout.");
 
+	@Documentation.Section(Documentation.Sections.ALL_TASK_MANAGER_NETWORK)
+	public static final ConfigOption<Integer> NETWORK_RETRIES =
+		key("taskmanager.network.retries")
```

Comment:
better to be consistent with the same prefix as others. `taskmanager.network.netty.client.connectRetries`

Reply:
@zhijiangW : https://github.com/apache/flink/pull/11541#discussion_r400969493

In the future we might want to re-use this property for other retries, not only client connection retries.

Reply:
that is fine for me